package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "node-sass",
   "version": "3.3.2",
-  "libsass": "3.2.5",
+  "libsass": "3.3.0",
```

Comment:
LibSass 3.3.0 can't go stable until atleast one RC has hit node-sass. This is because the vast majority of LibSass usage happens via node-sass and we use node-sass@beta release to weed out regressions.